.github/workflows/build-ci-container.yml

```diff
@@ -45,10 +45,10 @@ jobs:
         run: docker login -u $ACR_TOKEN_NAME -p ${{ secrets.ACR_CI_PUSH_TOKEN_PASSWORD }} $ACR_REGISTRY
 
       - name: Pull CI container
-        run: docker pull $ACR_REGISTRY/ccf/ci:02-02-2023-snp
+        run: docker pull $ACR_REGISTRY/ccf/ci:14-02-2023-snp
```

Comment:
This doesn't seem to be reflecting yet in GH actions, somewhat annoyingly. Not sure why.

Reply:
I suspect it's because https://github.com/microsoft/CCF/blob/main/.github/workflows/ci-containers.yml#L21 fetches tags from a cached copy of the repo.